### ISSUE TITLE
Parallel lines

### ISSUE THREAD

**Issue by aparzi** [openlayers/openlayers#14618]
is it possible to draw a line with this style?  Maybe taking advantage of the style object to make two parallel lines appear without creating two objects?

![RetteParallele1](https://user-images.githubusercontent.com/16490359/228042365-3a876501-c630-4c88-89fd-739978c4cd02.jpg)

**Comment by ahocevar** [comment]
Yes, that's possible using two styles, and one with its `geometry` set to the desired offset geometry.

**Comment by aparzi** [comment]
@ahocevar I'm going like this. I just wanted a confirmation if it was the right way.

Thanks

**Comment by aparzi** [comment]
Hi @ahocevar,
this solution works fine, but you have to calculate the angle correctly to apply the translated style. Because if I draw a line horizontally it is different from when I draw it vertically. I'm currently using Geometry's translate function to set a deltaX and deltaY. Any suggestions?

**Comment by mike-000** [comment]
You would need to use `Math.atan2` to calculate the angle and `Math.cos` and `Math.sin` to calculate offsets from that https://codesandbox.io/s/draw-features-forked-43fzw7?file=/main.js  (for multi-segment lines you might want to handle each segment separately)

**Comment by aparzi** [comment]
Thanks @mike-000, I solved.
